index.html

```diff
@@ -32,15 +31,195 @@
                         <h5>LET'S CONNECT</h5>
                     </div>
                 </div>
-                
-                <div class="social">
-                    <div class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></div>
-                    <div class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></div>
-                    <div class="social-media"><a href="#"><i class="fa fa-medium"></i></a></div>
-                    <div class="social-media"><a href="#"><i class="fa fa-github"></i></a></div>
-                    <div class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></div>
+                <ul class="social">
+                    <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
+                </ul>
+            </div>
+        </section>
+        <section class="works">
+            <div class="works-flex">
+                <div class="works-flex-item work-image">
+                    <img src="assets/images/portfolio_snapshoot/snapshoot1.png" class="img-style" alt="tonic works" />
+                </div>
+                <div class="works-flex-item">
+                    <h3 class="intro-text-works">Tonic</h3>
+                </div>
+                <ul class=" works-flex-item titles-container">
+                    <li class="list-main">CANOPY</li>
+                    <li class="titles-item">Back End Dev</li>
+                    <li class="titles-item">2015</li>
+                </ul>
+                <p class="works-flex-item works-description">
+                   A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="works-flex-item">
+                    <ul class="lang-container">
+                        <li class="lang">html</li>
+                        <li class="lang">css</li>
+                        <li class="lang">javascript</li>
+                    </ul>
```

Comment:
- [ ] Kudos to you for applying the changes. Taking a look at your design, I believe the changes caused some issues with the layout of the technologies inside your project cards. Let's try to fix the layout for all technologies in the project cards.

Your Design                                                        |  Figma Design                                |
:-------------------------------------------------------------:|:-----------------------------------------------:
![Your Design](https://user-images.githubusercontent.com/39523312/143503392-19aa8bc7-1c69-4b92-b788-3793f7d283fe.png)   |  ![Figma Design](https://user-images.githubusercontent.com/39523312/143503393-f915c80f-dd20-4ab7-af14-d09807e6e8cf.png)|